Index: elfutils-0.148/src/addr2line.c
===================================================================
--- elfutils-0.148.orig/src/addr2line.c	2013-12-09 14:41:03.060835726 +0000
+++ elfutils-0.148/src/addr2line.c	2013-12-09 14:41:04.016835740 +0000
@@ -447,10 +447,10 @@
       bool parsed = false;
       int i, j;
       char *name = NULL;
-      if (sscanf (string, "(%a[^)])%" PRIiMAX "%n", &name, &addr, &i) == 2
+      if (sscanf (string, "(%m[^)])%" PRIiMAX "%n", &name, &addr, &i) == 2
 	  && string[i] == '\0')
 	parsed = adjust_to_section (name, &addr, dwfl);
-      switch (sscanf (string, "%a[^-+]%n%" PRIiMAX "%n", &name, &i, &addr, &j))
+      switch (sscanf (string, "%m[^-+]%n%" PRIiMAX "%n", &name, &i, &addr, &j))
 	{
 	default:
 	  break;
Index: elfutils-0.148/tests/line2addr.c
===================================================================
--- elfutils-0.148.orig/tests/line2addr.c	2013-12-09 14:41:03.060835726 +0000
+++ elfutils-0.148/tests/line2addr.c	2013-12-09 15:04:14.792855813 +0000
@@ -132,7 +132,7 @@
     {
       struct args a = { .arg = argv[cnt] };
 
-      switch (sscanf (a.arg, "%a[^:]:%d", &a.file, &a.line))
+      switch (sscanf (a.arg, "%m[^:]:%d", &a.file, &a.line))
 	{
 	default:
 	case 0:
